raiden/tests/utils/eth_node.py

```diff
@@ -542,51 +528,33 @@ def run_private_blockchain(
             accounts_addresses=accounts_to_fund,
             seal_address=seal_account,
             random_marker=random_marker,
+            chain_id=chain_id,
         )
 
     elif blockchain_type == 'parity':
-        chainspec_path = f'{base_datadir}/chainspec.json'
+        genesis_path = f'{base_datadir}/chainspec.json'
         parity_generate_chain_spec(
-            spec_path=chainspec_path,
+            spec_path=genesis_path,
             accounts_addresses=accounts_to_fund,
             seal_account=seal_account,
             random_marker=random_marker,
+            chain_id=chain_id,
         )
-        parity_create_account(nodes_configuration[0], base_datadir, chainspec_path)
+        parity_create_account(nodes_configuration[0], base_datadir, genesis_path)
```

Comment:
Why are we doing this here?

Reply:
Yeah I was wondering that myself but didn't want to make the PR even bigger than it already is.
Ideally that should be moved to `eth_nodes_to_cmds` to be symmetrical to how we do it for geth.